src/yang/ietf-syslog.yang

```diff
@@ -700,69 +700,106 @@ module ietf-syslog {
+                             options.";
+                          reference
+                            "RFC 5426: Transmission of Syslog Messages
+                                       over UDP";
+                          list udp {
+                            key "address";
+                            description
+                              "List of all UDP sessions.";
+                            leaf address {
+                              type inet:host;
+                              description
+                                "The leaf uniquely specifies the
+                                 address of the remote host. One of
+                                 the following must be specified: an
+                                 ipv4 address, an ipv6 address, or a
+                                 host name.";
+                            }
+                            leaf port {
+                              type inet:port-number;
+                              default "514";
+                              description
+                                "This leaf specifies the port number
+                                 used to deliver messages to the remote
+                                 server.";
                             }
+                          }
                         }
-                        case tls {
-                            container tls {
-                                description
-                                  "This container describes the TLS
-                                   transport options.";
-                                reference
-                                  "RFC 5425: Transport Layer Security
-                                   (TLS) Transport Mapping for
-                                   Syslog ";
-                                leaf address {
-                                    type inet:host;
-                                    description
-                                      "The leaf uniquely specifies
-                                       the address of the remote
-                                       host. One of the following
-                                       must be specified: an ipv4
-                                       address, an ipv6 address, or
-                                       a host name.";
-                                }
-                                leaf port {
-                                    type inet:port-number;
-                                    default "6514";
-                                    description
-                                      "TCP port 6514 has been
-                                       allocated as the default port
-                                       for syslog over TLS.";
-                                }
-                                uses tlsc:tls-client-grouping;
+                      }
+                      case tcp {
```

Comment:
Nice!  I think we might want some text on this in the draft itself.  We might need to update security considerations, too.